extensions/ql-vscode/src/local-queries/local-queries.ts

```diff
@@ -48,7 +48,7 @@ import { SkeletonQueryWizard } from "../skeleton-query-wizard";
 import { LocalQueryRun } from "./local-query-run";
 import { createMultiSelectionCommand } from "../common/vscode/selection-commands";
 import { findLanguage } from "../codeql-cli/query-language";
-import type { QueryTreeViewItem } from "../queries-panel/query-tree-view-item";
+import { QueryTreeViewItem } from "../queries-panel/query-tree-view-item";
```

Comment:
Is there a reason you removed the `type` import specification?

Reply:
Good spot. Earlier in the lifetime of this PR I had to convert the `type` import because I used `QueryTreeViewItem` in a `instanceof` expression, and that requires importing the type at runtime instead of just at compiletime. But that has been removed now so this can stay as a `type` import.